site/topics/control-logic/instructions-microcodes.rst

```diff
@@ -18,6 +18,74 @@ Instructions and Microcodes
 Constraints
 ===========
 
+* The designers of the computer (us) decide what instructions to include and how they are to be performed
+* However, it is important to consider the limitations and constraints of the system
+
+* Although the ESAP system is to be Turing Complete, it is limited by the fact that it's an 8 bit system
```

Comment:
Probably drop _Turing complete_ -- they won't know what that means for another two years. Maybe _...is a fully functional computer..._